src/pywbem/twisted_client.py

```diff
@@ -300,6 +302,109 @@ def parseResponse(self, xml):
         return [pywbem.tupleparse.parse_instance(x) for x in tt]
 
 
+class OpenEnumerateInstances(WBEMClientFactory):
+    """Factory to produce EnumerateInstances WBEM clients."""
+
+    def __init__(self, creds, classname, namespace='root/cimv2', **kwargs):
```

Comment:
Look at my comment above regarding the 'root/cimv2' namespace.

Reply:
As I understand it is default style for this library, in that case, we need to change all `namespace='root/cimv2'` in this library.